package.json

```diff
@@ -13,34 +13,37 @@
     "watch:js": "cross-env BABEL_ENV=quickstart node ./scripts/js/babel-es6-watch.js"
   },
   "devDependencies": {
+    "@babel/core": "^7.0.0",
```

Comment:
In testing I ran into some issues with `yarn watch` not working. With the change here from `babel-core` to `@babel/core` it looks like the following change is necessary in line 3 of `scripts/js/compile.js`:
`const babel = require('@babel/core');`

Reply:
Good catch.  Thanks @tadean!